scripts/commands/build.js

```diff
@@ -62,17 +56,18 @@ async function buildRelease( currentBranchName ) {
 	);
 
 	if ( ! shouldContinue ) {
-		info( 'Aborting release build.' );
-		await switchToBranchWithMessage( currentBranchName );
-
-		return false;
+		return abortAndSwitchToBranch(
+			'Aborting release build.',
+			'info',
+			currentBranchName
+		);
 	}
 
 	info( 'Creating a new release build. This make take some time...' );
 	const interval = setInterval( () => process.stdout.write( '.' ), 1000 );
 
 	await execAsync( 'npm i' );
```

Comment:
Should we run `npm ci` here to make sure we're installing the full set of dependencies and don't have any unexpected additions/deviations? See the [npm-ci docs](https://docs.npmjs.com/cli/v8/commands/npm-ci) for more details.